task09-reactjs/task09/src/components/General-info.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * All Rights Reserved
+ * This software is property information of AKUREY S.A.
+ * Contact us at contact@akurey.com
+ * @summary js for the main-info-container-component
+ */
+
+import { useEffect, useState } from 'react';
+import '../scss/Body.scss';
+   
+
+ function GeneralInfo() {
+    const [ info, setInfo ] = useState(null);
+    
+    const fetchOnePieceInfo = async () => {
+        const url = 'https://static.akurey.com/trainings/OnePieceInformation.json';
+        try{
+             const onePieceInfo = await fetch(url)
+             const response = await onePieceInfo.json();
+
+             sortArray(response.islands);
+             sortArray(response.characters);
+             sortArray(response.mysticObjects);
+             setInfo(response);
+        } catch (error){
+         alert(error)
+        }
+     }
+
+    useEffect(()=>{
+        fetchOnePieceInfo();
+    },[])
+
+   return (
+    <div className="main-info-container">
+        <form>
+        <div className="col-12">
+            <div className="input-group w-90 mx-auto">
+            <div className="input-group-prepend">
+                <span className="input-group-text"><i className="icon-search"></i></span>
+            </div>
+            <input type="text" className="form-control search-form-control"
+                placeholder="Search Characters, Islands or Objects" />
+            </div>
+        </div>
+        </form>
+        <div className="main-photos-container">
+            <div className="characters-container">
+                <div className="titles-container">
+                    <p className="section-titles">Characters</p>
+                    <a href="/characters" className="see-all-titles">See all</a>
+                </div>
+                <div className="characters-photo-container" id="characters-photo-container">
```

Comment:
I think this warning comes from this "maps"
<img width="828" alt="image" src="https://user-images.githubusercontent.com/48458005/186730973-dbd140ca-3298-44fe-9042-0536fb899adf.png">
Please check the warning ( https://reactjs.org/docs/lists-and-keys.html#keys)